src/lib/utils.js

```diff
@@ -65,13 +65,39 @@ export const loadStorage = () => {
   }
 }
 
+/**
+ * Returns a object with RGBA.
+ * @param {string} value RGBA
+ * https://stackoverflow.com/a/11003212/11321732
+ */
+const getColorObject = value => {
+  if ( [ 'none', 'undefined', null ].includes( value ) ) return null
+  const rgba = value.match( /\d+/g )
+  const rgbaObject = { r: rgba[0], g: rgba[1], b: rgba[2], a: rgba[3] }
+  return rgbaObject
+}
+
+const writeCssToDom = ( key, value ) => document.documentElement.style.setProperty( key, value )
+
+/**
+ * Writes values for CSS to the DOM
+ * @param {string} key CSS variable to update.
+ * @param {string} value new value.
+ */
+export const writeCss = ( key, value ) => {
+  // Drop colors require to be string https://github.com/ShabadOS/desktop/blob/fc01ec563178ad605e42c2274b030da366063a13/app/frontend/src/Overlay/themes/Example.template#L82
+  if ( key.includes( 'drop' ) ) {
+    const rgba = getColorObject( value )
+    if ( rgba === null ) writeCssToDom( key, value )
+    else writeCssToDom( key, `${rgba.r}, ${rgba.g}, ${rgba.b}` )
+  } else writeCssToDom( key, localStorage[key] )
```

Comment:
It took me a while to understand what `key.includes( 'drop' )` actually means. I would prefer this to be handled in a more explicit manner, through declaration as an additional parameter in `options`.

Logic can be made more readable. Consider using early returns to reduce level of nesting, and reduce the calls to `writeCssToDom`, which appears to be superfluous. 

```suggestion
  if ( key.includes( 'drop' ) ) {
    const rgba = getColorObject( value )
    writeCssToDom( key, rgba ? `${rgba.r}, ${rgba.g}, ${rgba.b}` : value )
    
    return
  }
  
  writeCssToDom( key, localStorage[key] )
```

Reply:
I can change 'drop' to something more exact like 'drop-color'

Reply:
It's more that I don't think the idea of doing this based on part of the property name is robust, it's the kind of thing that might bite you in the future inadvertently

Reply:
But the options are not always being passed down. In case of loading from local storage the css variable is used. I can make it more explicit by just making sure that only those two specific drop colors are changed. 

```
  if ( [ OPTIONS.primaryDropColor.storageKey, OPTIONS.secondaryDropColor.storageKey ].includes( key ) ) {
    const rgba = getColorObject( value )
    writeCssToDom( key, rgba ? `${rgba.r}, ${rgba.g}, ${rgba.b}` : value )
  } else writeCssToDom( key, localStorage[key] )
```

Reply:
Whitelisting is also ok.

Reply:
I'd still suggest early-return + scrapping the `else`, as above. Less compact and more readable.